test/functional/api/vim_spec.lua

```diff
@@ -34,6 +34,84 @@ describe('API', function()
                request, nil)
   end)
 
+  describe('nvim_source', function()
+	it('works with a one-liner', function()
```

Comment:
please indent the it() cases in the describe() block, as in the rest of the file.